src/dodal/devices/fast_grid_scan.py

```diff
@@ -208,35 +209,38 @@ def clean_up(self):
 class FastGridScan(Device):
     x_steps: EpicsSignalWithRBV = Component(EpicsSignalWithRBV, "X_NUM_STEPS")
     y_steps: EpicsSignalWithRBV = Component(EpicsSignalWithRBV, "Y_NUM_STEPS")
-    z_steps: EpicsSignalWithRBV = Component(EpicsSignalWithRBV, "Z_NUM_STEPS")
+    #z_steps: EpicsSignalWithRBV = Component(EpicsSignalWithRBV, "Z_NUM_STEPS")
+    z_steps: Signal = Component(Signal, kind=Kind.hinted, value=1)
 
     x_step_size: EpicsSignalWithRBV = Component(EpicsSignalWithRBV, "X_STEP_SIZE")
     y_step_size: EpicsSignalWithRBV = Component(EpicsSignalWithRBV, "Y_STEP_SIZE")
-    z_step_size: EpicsSignalWithRBV = Component(EpicsSignalWithRBV, "Z_STEP_SIZE")
+    # z_step_size: EpicsSignalWithRBV = Component(EpicsSignalWithRBV, "Z_STEP_SIZE")
+    z_step_size: Signal = Component(Signal, kind=Kind.hinted, value=1)
 
-    dwell_time: EpicsSignalWithRBV = Component(EpicsSignalWithRBV, "DWELL_TIME")
+    dwell_time: EpicsSignalWithRBV = Component(EpicsSignalWithRBV, "EXPOSURE_TIME")
 
     x_start: EpicsSignalWithRBV = Component(EpicsSignalWithRBV, "X_START")
     y1_start: EpicsSignalWithRBV = Component(EpicsSignalWithRBV, "Y_START")
-    y2_start: EpicsSignalWithRBV = Component(EpicsSignalWithRBV, "Y2_START")
+    # y2_start: EpicsSignalWithRBV = Component(EpicsSignalWithRBV, "Y2_START")
     z1_start: EpicsSignalWithRBV = Component(EpicsSignalWithRBV, "Z_START")
-    z2_start: EpicsSignalWithRBV = Component(EpicsSignalWithRBV, "Z2_START")
+    # z2_start: EpicsSignalWithRBV = Component(EpicsSignalWithRBV, "Z2_START")
 
     position_counter: EpicsSignal = Component(
-        EpicsSignal, "POS_COUNTER", write_pv="POS_COUNTER_WRITE"
+        EpicsSignal, "POS_COUNTER_RBV", write_pv="POS_COUNTER"
     )
     x_counter: EpicsSignalRO = Component(EpicsSignalRO, "X_COUNTER")
     y_counter: EpicsSignalRO = Component(EpicsSignalRO, "Y_COUNTER")
-    scan_invalid: EpicsSignalRO = Component(EpicsSignalRO, "SCAN_INVALID")
+    # scan_invalid: EpicsSignalRO = Component(EpicsSignalRO, "SCAN_INVALID")
+    scan_invalid: Signal = Component(Signal, kind=Kind.hinted, value=0)
 
     run_cmd: EpicsSignal = Component(EpicsSignal, "RUN.PROC")
     stop_cmd: EpicsSignal = Component(EpicsSignal, "STOP.PROC")
-    status: EpicsSignalRO = Component(EpicsSignalRO, "SCAN_STATUS")
+    status: EpicsSignalRO = Component(EpicsSignalRO, "SCAN_STATUS_RBV")
 
     expected_images: Signal = Component(Signal)
 
     # Kickoff timeout in seconds
-    KICKOFF_TIMEOUT: float = 5.0
+    KICKOFF_TIMEOUT: float = 20.0
```

Comment:
Is it possible to make this generic/configurable?

Also we might be able to do this better, the main delay is waiting for completion callbacks on each parameter we set, if we set them all and then wait for all callbacks in parallel it may be significantly faster. Testing with Andy showed a 0.5-1s delay per parameter.